composer.json

```diff
@@ -31,7 +31,7 @@
 		"ext-libxml": "*",
 		"ext-mbstring": "*",
 		"ext-simplexml": "*",
-		"league/flysystem": "^1.0|^2.0|^3.0",
+		"league/flysystem": "^1.0|2.1.1|^3.0",
```

Comment:
Why did you restricted tho this version???
Why not ^2.1 or ~2.1.1 if you do not want to upgrade the features?